background/transport.js

```diff
@@ -24,6 +24,10 @@ let transport = null;
 // Attempt to extract token on start if not available.
 extractToken();
 
+export const getTransport = () => {
```

Comment:
I think it would be better to refactor all `modules` into class definitions and bootstrap application just in a single place and avoid treating any module as a singleton.
It's especially important since we have many different contexts, and just calling `getTransport()` in some context might not work at all.